apps/io-func-wallet-solution-backend/src/infra/attestation-service/ios/attestation.ts

```diff
@@ -0,0 +1,166 @@
+import { createHash, X509Certificate } from "crypto";
+import * as asn1js from "asn1js";
+import * as pkijs from "pkijs";
+import * as jose from "jose";
+
+import { iOsAttestation } from ".";
+
+const APPATTESTDEVELOP = Buffer.from("appattestdevelop").toString("hex");
+const APPATTESTPROD = Buffer.concat([
+  Buffer.from("appattest"),
+  Buffer.from([0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]),
+]).toString("hex");
+
+export type VerifyAttestationParams = {
+  decodedAttestation: iOsAttestation;
+  challenge: string;
+  keyId: string;
+  bundleIdentifier: string;
+  teamIdentifier: string;
+  allowDevelopmentEnvironment: boolean;
+  appleRootCertificate: string;
+};
+
+/* This function does not lend itself well to functional programming,
+ * therefore the validation was written in imperative language
+ */
```

Comment:
This is a personal thought on the implementation. What do you think to remove it or replace it with typical comment
  
```
/**
* function description
* @param {string} param - param description
*/
```

In general, what do you think about using a multi-line comment for the functions header description as a best practice?

Reply:
yes, agreed. I'll try to do a small refactor with a dedicated PR. I'll delete the comment for now